spec/peep_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+describe Peep do
+  it 'it expects to be created and retrieved from the database' do
+    user = User.create(name: 'Rob',
```

Comment:
Still investigating how to effectively stub this out, the datamapper save_parents method is causing me a problem as I cannot find a way to instigate the relationship using a double, I can respond to the command, but not get the code doing what is should, which is sort of making me think that if I achieved this I would be testing the complexity of the stub rather than the code in peep?